osu.Game.Tournament/Components/TournamentBeatmapPanel.cs

```diff
@@ -22,13 +22,13 @@ public partial class TournamentBeatmapPanel : CompositeDrawable
     {
         public readonly TournamentBeatmap? Beatmap;
 
-        private readonly string mod;
+        private readonly string? mod;
```

Comment:
actively wrong, `mod` will never be null because of the ctor default value spec

https://github.com/ppy/osu/blob/55cb400674c23656bc6cde9345c2871002380bd4/osu.Game.Tournament/Components/TournamentBeatmapPanel.cs#L33-L36

Reply:
ok